packages/patternfly-3/patternfly-react-extensions/src/components/TableGrid/TableGridColumnHeader.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Button, Grid, Icon, noop } from 'patternfly-react';
+
+/**
+ * TableGridColumnHeader Component for PatternFly
+ */
+
+const TableGridColumnHeader = ({ children, className, sortable, isSorted, isAscending, onSortToggle, ...props }) => {
+  const classes = classNames(
+    'table-grid-pf-column-header text-nowrap',
+    { 'active-sort': isSorted, descending: !isAscending },
+    className
+  );
+
+  return (
+    <Grid.Col className={classes} {...props}>
+      {sortable && (
+        <React.Fragment>
```

Comment:
Not sure you need the ` <React.Fragment>` here.

Reply:
Ah, correct. Leftover...